certbot/tests/main_test.py

```diff
@@ -1023,8 +1023,9 @@ def test_certonly_new_request_failure(self):
 
     def _test_renewal_common(self, due_for_renewal, extra_args, log_out=None,
                              args=None, should_renew=True, error_expected=False,
-                                 quiet_mode=False, expiry_date=datetime.datetime.now()):
-        # pylint: disable=too-many-locals,too-many-arguments
+                             quiet_mode=False, expiry_date=datetime.datetime.now(),
+                             reuse_key=False):
+        # pylint: disable=too-many-locals,too-many-arguments,too-many-branches
```

Comment:
Is there a way to add the tests that doesn't stick the checks into `_test_renewal_common`? We already overload this with too much stuff.

Reply:
I agree, but I think it might look even more complex by defining a whole new set of ways to test renewals!